### PR TITLE
docs: Add lua filter example

### PR DESCRIPTION
**Pull request by francescsanjuanmrf**

### CODE REVIEW

File: pipeline/filters/lua.md [hunk 1]
```diff
@@ -262,3 +262,86 @@ end
 ```
 
 See also [Fluent Bit: PR 811](https://github.com/fluent/fluent-bit/pull/811).
+
+### Responde code filtering
```
Comment (iagoalonsomrf) [inline pipeline/filters/lua.md:266]: Typo :)

File: pipeline/filters/lua.md [hunk 2]
```diff
@@ -262,3 +262,86 @@ end
...
+    return 0,0,0
+  end
+end
+```
+
+#### Configuration
+
+Configuration to get istio logs and apply response code filter to them.
+
+```python
```
Comment (iagoalonsomrf) [inline pipeline/filters/lua.md:292]: Shouldn't matter much, but why does this have a `python` format?

File: pipeline/filters/lua.md [hunk 3]
```diff
@@ -262,3 +262,86 @@ end
...
+            "response_flags":"-",
+            "start_time":"2022-07-28T11:16:51.663Z",
+            "upstream_transport_failure_reason":null,
+            "method":"POST","connection_termination_details":null,
+            "protocol":"HTTP/1.1","requested_server_name":null,
+            "upstream_service_time":"6161"
+        },
+    "stream":"stdout",
+    "time":"2022-07-28T11:17:06.704109897Z"
+}
```
Comment (iagoalonsomrf) [inline pipeline/filters/lua.md:342]: I would also apply formatting to this json.

```json
{
    "log": {
        "response_code": 200,
        "bytes_sent": 111328341,
        "authority": "randomservice.randomservice",
        "duration": 14493,
        "request_id": "2e9d38f8-36a9-40a6-bdb2-47c8eb7d399d",
        "upstream_local_address": "10.11.82.178:42738",
        "downstream_local_address": "10.10.21.17:80",
        "upstream_cluster": "outbound|80||randomservice.svc.cluster.local",
        "x_forwarded_for": null,
        "route_name": "default",
        "upstream_host": "10.11.6.90:80",
        "user_agent": "RandomUserAgent",
        "response_code_details": "via_upstream",
        "downstream_remote_address": "10.11.82.178:51096",
        "bytes_received": 1148,
        "path": "/?parameter=random",
        "response_flags": "-",
        "start_time": "2022-07-28T11:16:51.663Z",
        "upstream_transport_failure_reason": null,
        "method": "POST",
        "connection_termination_details": null,
        "protocol": "HTTP/1.1",
        "requested_server_name": null,
        "upstream_service_time": "6161"
    },
    "stream": "stdout",
    "time": "2022-07-28T11:17:06.704109897Z"
}
```

**Review by joantomas — APPROVED** [review]

File: pipeline/filters/lua.md (hunk 1 above)
Comment (francescsanjuanmrf) [reply pipeline/filters/lua.md:266]: Fixed

File: pipeline/filters/lua.md (hunk 2 above)
Comment (francescsanjuanmrf) [reply pipeline/filters/lua.md:292]: Changed to ini :)

File: pipeline/filters/lua.md (hunk 3 above)
Comment (francescsanjuanmrf) [reply pipeline/filters/lua.md]: better fromat

**Pull request merged**